pymisp/api.py

```diff
@@ -1243,9 +1243,12 @@ def _set_organisation_parameters(self, **kwargs):
             organisation['local'] = kwargs.get('local')
         return organisation
 
-    def get_organisations_list(self):
+    def get_organisations_list(self, scope="local"):
         session = self.__prepare_session()
-        url = urljoin(self.root_url, 'organisations')
+        scope=scope.lower()
+        if scope not in ["local", "external", "all"]:
```

Comment:
Worth raising a `ValueError` here instead of resetting to default - avoids anyone accidentally entering `externl` and wondering why it's always pulling local ;)